utils/grafana-container.bash

```diff
@@ -0,0 +1,12 @@
+set -ex
+docker rm -f grafana || true
```

Comment:
we should use a more unique name (`grafana--tribe-29-checkmk-datasource`?), otherwise we could delete containers without warning.